Resources/Prototypes/Recipes/Lathes/Packs/shared.yml

```
@@ -50,6 +50,8 @@
   - ClothingMaskWeldingGas
   - SignallerAdvanced
   - BluespaceAirTank
```

Aaaand...

```suggestion
  - BluespaceAirTank # Frontier
```
Since you're the one who added this on the *other* bluespace tank PR, could you fix this, pretty please ?